src/test/java/org/fogbowcloud/manager/core/plugins/behavior/federationidentity/ldap/LdapLocalIdentityPluginTest.java

```diff
@@ -23,121 +25,83 @@
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_BASE, "ldapBase");
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_ENCRYPT, "");
-        userCredentials.put(LdapIdentityPlugin.CRED_PRIVATE_KEY, "private_key_path");
-        userCredentials.put(LdapIdentityPlugin.CRED_PUBLIC_KEY, "public_key_path");
-
+    	//set up
         Mockito.doThrow(new Exception("Invalid User"))
-                .when(ldapStoneIdentity)
-                .ldapAuthenticate(Mockito.eq(name), Mockito.eq(password));
-        try {
-            ldapStoneIdentity.createFederationTokenValue(userCredentials);
-            Assert.fail();
-        } catch (UnauthenticatedUserException e) {
-            Assert.assertEquals("Couldn't load account summary from LDAP Server.", e.getMessage());
-        }
+                .when(this.ldapStoneIdentity)
+                .ldapAuthenticate(Mockito.eq(name), Mockito.eq(this.password));
+        
+        //exercise/verify
+        this.ldapStoneIdentity.createFederationTokenValue(this.userCredentials);
     }
-
+    
+    //test case: check if isValid returns false when the accessId is invalid.
     @Test
     public void testGetTokenInvalidAccessId() throws Exception {
-
-        String name = "ldapUser";
-        String password = "ldapUserPass";
-        String userName = "User Full Name";
-
-        Map<String, String> userCredentials = new HashMap<String, String>();
-        userCredentials.put(LdapIdentityPlugin.CRED_USERNAME, name);
-        userCredentials.put(LdapIdentityPlugin.CRED_PASSWORD, password);
-        userCredentials.put(LdapIdentityPlugin.CRED_AUTH_URL, "ldapUrl");
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_BASE, "ldapBase");
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_ENCRYPT, "");
-        userCredentials.put(LdapIdentityPlugin.CRED_PRIVATE_KEY, "private_key_path");
-        userCredentials.put(LdapIdentityPlugin.CRED_PUBLIC_KEY, "public_key_path");
-
-        Mockito.doReturn(userName)
+    	//set up
+        Mockito.doReturn(this.userName)
                 .when(this.ldapStoneIdentity)
-                .ldapAuthenticate(Mockito.eq(name), Mockito.eq(password));
-
-        String tokenValueA = this.ldapStoneIdentity.createFederationTokenValue(userCredentials);
-
+                .ldapAuthenticate(Mockito.eq(this.name), Mockito.eq(this.password));
+        String tokenValueA = this.ldapStoneIdentity.createFederationTokenValue(this.userCredentials);
         String decodedAccessId = decodeAccessId(tokenValueA);
-
         String split[] = decodedAccessId.split(LdapIdentityPlugin.ACCESSID_SEPARATOR);
-        String tokenMessage = split[0];
         String signature = split[1];
-
         String newAccessId =
-                "{name:\"nome\", expirationDate:\"123421\"}"
+                "{name:\"nome\", expirationDate:\"" + (new Date(new Date().getTime() + TimeUnit.DAYS.toMillis(365))).getTime() + "\"}"
```

Comment:
This seems hard to read at first. Can you extract it to a variable with a meaningful name?

Reply:
As you wish.